### PR TITLE
feat(doc): introduce global tolerations and nodeselectors for user deployed components

### PR DESCRIPTION
**Pull request by ChanYiLin**
ref: https://github.com/longhorn/longhorn/issues/5089

- Add global tolerations and nodeselectors for user deployed components
- Use global setting by default
- Override each by their specific settings .

### CODE REVIEW

File: content/docs/1.7.0/advanced-resources/deploy/node-selector.md [hunk 1]
```diff
@@ -15,7 +15,13 @@ You need to set node selector for both types of components. See more details bel
 
 ### Setting up Node Selector During installing Longhorn
 1. Set node selector for user deployed components (Manager, UI, Driver Deployer)
-   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML:
+   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML to apply the values to all the components at once:
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/node-selector.md:18]:
```suggestion
   * If you install Longhorn through Rancher 2.5.x, you must copy and paste the following parameters into the YAML on the Rancher UI (**SCREEN_TITLE** > **Edit as YAML**) to apply the value to all user-deployed components.
```

File: content/docs/1.7.0/advanced-resources/deploy/node-selector.md [hunk 2]
```diff
@@ -15,7 +15,13 @@ You need to set node selector for both types of components. See more details bel
...
+      ```yaml
+        global:
+          nodeSelector:
+            label-key1: "label-value1"
+      ```
+   * You can also specify the node selectors for each user deployed components:
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/node-selector.md:24]:
```suggestion
   * You can also specify the node selector for each user-deployed component.
```

File: content/docs/1.7.0/advanced-resources/deploy/node-selector.md [hunk 3]
```diff
@@ -29,7 +35,7 @@ You need to set node selector for both types of components. See more details bel
       ```
    * If you install Longhorn by using `kubectl` to apply [the deployment YAML](https://raw.githubusercontent.com/longhorn/longhorn/v1.1.1/deploy/longhorn.yaml), you need to modify the node selector section for Longhorn Manager, Longhorn UI, and Longhorn Driver Deployer.
     Then apply the YAMl files.
-   * If you install Longhorn using Helm, you can change the Helm values for `longhornManager.nodeSelector`, `longhornUI.nodeSelector`, `longhornDriver.nodeSelector` in the `values.yaml` file.
+   * If you install Longhorn using Helm, you can change the Helm values for `global.nodeSelector`, `longhornManager.nodeSelector`, `longhornUI.nodeSelector`, `longhornDriver.nodeSelector` in the `values.yaml` file.
     Then install the chart
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/node-selector.md:39]:
```suggestion
   * If you install Longhorn using Helm, you can change the Helm values for `global.nodeSelector`, `longhornManager.nodeSelector`, `longhornUI.nodeSelector`, `longhornDriver.nodeSelector` in the `values.yaml` file before installing the chart.
```

File: content/docs/1.7.0/advanced-resources/deploy/node-selector.md [hunk 4]
```diff
@@ -55,7 +61,13 @@ You need to set node selector for both types of components. See more details bel
    * To ensure that your preferred settings are immediately applied, stop all workloads and detach all Longhorn volumes before applying it.
 
 2. Set node selector for user deployed components
-    * If you install Longhorn by Rancher UI, you need to click `Edit as YAML` and copy this values into the YAML then click upgrade:
+   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML to apply the values to all the components at once:
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/node-selector.md:64]:
```suggestion
   * If you install Longhorn through Rancher 2.5.x, you must copy and paste the following parameters into the YAML on the Rancher UI (**SCREEN_TITLE** > **Edit as YAML**) to apply the value to all system-managed components.
```

File: content/docs/1.7.0/advanced-resources/deploy/node-selector.md [hunk 5]
```diff
@@ -55,7 +61,13 @@ You need to set node selector for both types of components. See more details bel
...
+        ```yaml
+        global:
+          nodeSelector:
+            label-key1: "label-value1"
+        ```
+    * You can also specify the node selectors for each user deployed components:
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/node-selector.md:70]:
```suggestion
    * You can also specify the node selector for each system-managed component.
```

File: content/docs/1.7.0/advanced-resources/deploy/node-selector.md [hunk 6]
```diff
@@ -69,7 +81,7 @@ You need to set node selector for both types of components. See more details bel
         ```
     * If you install Longhorn by using `kubectl` to apply [the deployment YAML](https://raw.githubusercontent.com/longhorn/longhorn/v1.1.1/deploy/longhorn.yaml), you need to modify the node selector section for Longhorn Manager, Longhorn UI, and Longhorn Driver Deployer.
       Then reapply the YAMl files.
-    * If you install Longhorn using Helm, you can change the Helm values for `longhornManager.nodeSelector`, `longhornUI.nodeSelector`, `longhornDriverDeployer.nodeSelector` in the `value.yaml` file.
+    * If you install Longhorn using Helm, you can change the Helm values for `global.nodeSelector`, `longhornManager.nodeSelector`, `longhornUI.nodeSelector`, `longhornDriverDeployer.nodeSelector` in the `value.yaml` file.
       Then do Helm upgrade the chart.
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/node-selector.md:85]:
```suggestion
    * If you install Longhorn using Helm, you can change the Helm values for `global.nodeSelector`, `longhornManager.nodeSelector`, `longhornUI.nodeSelector`, `longhornDriverDeployer.nodeSelector` in the `values.yaml` file, and then run `helm upgrade` to upgrade to the new version of the chart.
```

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 7]
```diff
@@ -15,7 +15,16 @@ You need to set tolerations for both types of components. See more details below
 
 ### Setting up Taints and Tolerations During installing Longhorn
 1. Set taint tolerations for user deployed components (Manager, UI, Driver Deployer)
-   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML:
+   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML to apply the values to all the components at once:
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:18]:
```suggestion
   * If you install Longhorn through Rancher 2.5.x, you must copy and paste the following parameters into the YAML on the Rancher UI (**SCREEN_TITLE** > **Edit as YAML**) to apply the value to all user-deployed components.
```

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 8]
```diff
@@ -15,7 +15,16 @@ You need to set tolerations for both types of components. See more details below
...
+     ```yaml
+       global:
+         tolerations:
+         - key: "key"
+           operator: "Equal"
+           value: "value"
+           effect: "NoSchedule"
+    ```
+   * You can also specify the tolerations for each user deployed components:
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:27]:
```suggestion
   * You can also specify the tolerations for each user-deployed component.
```

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 9]
```diff
@@ -38,7 +47,7 @@ You need to set tolerations for both types of components. See more details below
      ```
    * If you install Longhorn by using `kubectl` to apply [the deployment YAML](https://raw.githubusercontent.com/longhorn/longhorn/v1.1.1/deploy/longhorn.yaml), you need to modify the taint tolerations section for Longhorn Manager, Longhorn UI, and Longhorn Driver Deployer.
     Then apply the YAMl files.
-   * If you install Longhorn using Helm, you can change the Helm values for `longhornManager.tolerations`, `longhornUI.tolerations`, `longhornDriver.tolerations` in the `values.yaml` file.
+   * If you install Longhorn using Helm, you can change the Helm values for `global.tolerations`, `longhornManager.tolerations`, `longhornUI.tolerations`, `longhornDriver.tolerations` in the `values.yaml` file.
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:50]:
```suggestion
   * If you install Longhorn using Helm, you can change the Helm values for `global.tolerations`, `longhornManager.tolerations`, `longhornUI.tolerations`, `longhornDriver.tolerations` in the `values.yaml` file before installing the chart.
```

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 10]
```diff
@@ -70,7 +79,16 @@ You need to set tolerations for both types of components. See more details below
    To ensure that your preferred settings are immediately applied, stop all workloads and detach all Longhorn volumes before configuring the settings.
 
 2. Set taint tolerations for user deployed components (Manager, UI, Driver Deployer)
-   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML:
+   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML to apply the values to all the components at once:
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:82]:
```suggestion
   * If you install Longhorn through Rancher 2.5.x, you must copy and paste the following parameters into the YAML on the Rancher UI (**SCREEN_TITLE** > **Edit as YAML**) to apply the value to all system-managed components.
```

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 11]
```diff
@@ -70,7 +79,16 @@ You need to set tolerations for both types of components. See more details below
...
+     ```yaml
+       global:
+         tolerations:
+         - key: "key"
+           operator: "Equal"
+           value: "value"
+           effect: "NoSchedule"
+    ```
+   * You can also specify the tolerations for each user deployed components:
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:91]:
```suggestion
   * You can also specify the tolerations for each system-managed component.
```

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 12]
```diff
@@ -93,7 +111,7 @@ You need to set tolerations for both types of components. See more details below
      ```
    * If you install Longhorn by using `kubectl` to apply [the deployment YAML](https://raw.githubusercontent.com/longhorn/longhorn/v1.1.1/deploy/longhorn.yaml), you need to modify the taint tolerations section for Longhorn Manager, Longhorn UI, and Longhorn Driver Deployer.
   Then reapply the YAMl files.
-   * If you install Longhorn using Helm, you can change the Helm values for `longhornManager.tolerations`, `longhornUI.tolerations`, `longhornDriver.tolerations` in the `values.yaml` file.
+   * If you install Longhorn using Helm, you can change the Helm values for `global.tolerations`, `longhornManager.tolerations`, `longhornUI.tolerations`, `longhornDriver.tolerations` in the `values.yaml` file.
   Then do Helm upgrade the chart.
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:115]:
```suggestion
   * If you install Longhorn using Helm, you can change the Helm values for `global.tolerations`, `longhornManager.tolerations`, `longhornUI.tolerations`, `longhornDriver.tolerations` in the `values.yaml` file, and then run `helm upgrade` to upgrade to the new version of the chart.
```

File: content/docs/1.7.0/advanced-resources/deploy/node-selector.md (hunk 4 above)
Comment (jillian-maroket) [reply content/docs/1.7.0/advanced-resources/deploy/node-selector.md:64]: Can we specify the part of the Rancher UI where users can find the **Edit as YAML** button?

**Review by jillian-maroket — COMMENTED** [review]
@ChanYiLin Please see my suggestions and question.

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md (hunk 11 above)
Comment (ChanYiLin) [reply content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:91]: its not "system-managed", it is still "user-deployed" here

The structure of the article is

- Install
    - user-deployed
    - system-managed
- upgrade
    - user-deployed
    - system-managed 

All I changed is only "user-deployed" in "install" and "upgrade" part

**Comment by ChanYiLin** [comment]
> @ChanYiLin I have completed the review. Can we specify where users can find the **Edit as YAML** button on the Rancher UI?

It doesn't locate at some fixed place, it appears during the process of the installationa/upgrade
https://www.virtualizationhowto.com/2022/05/install-longhorn-rancher-for-kubernetes-block-storage/
You can check the process of the installation, it appears in the "Install: Step 2" **which asks users to configure the helm value**.

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 13]
```diff
@@ -10,12 +10,21 @@ Notice that the taint tolerations setting for one workload will not prevent it f
 For more information about how taints and tolerations work, refer to the [official Kubernetes documentation.](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
 
 # Setting up Taints and Tolerations
-Longhorn system contains user deployed components (e.g, Manager, Driver Deployer, UI) and system managed components (e.g, Instance Manager, Engine Image, CSI Driver, etc.)
+Longhorn system contains user-deployed components (e.g, Manager, Driver Deployer, UI) and system-managed components (e.g, Instance Manager, Engine Image, CSI Driver, etc.)
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:13]:
```suggestion
Longhorn consists of user-deployed components (for example, Longhorn Manager, Longhorn Driver, and Longhorn UI) and system-managed components (for example, Instance Manager, Backing Image Manager, Share Manager, CSI Driver, and Engine Image). 
```

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 14]
```diff
@@ -10,12 +10,21 @@ Notice that the taint tolerations setting for one workload will not prevent it f
...
 You need to set tolerations for both types of components. See more details below.
 
 ### Setting up Taints and Tolerations During installing Longhorn
-1. Set taint tolerations for user deployed components (Manager, UI, Driver Deployer)
-   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML:
+1. Set taint tolerations for user-deployed components (Manager, UI, Driver Deployer)
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:17]:
```suggestion
1. Set taint tolerations for user-deployed components (for example, Longhorn Manager, Longhorn Driver, and Longhorn UI).
```

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 15]
```diff
@@ -38,7 +47,7 @@ You need to set tolerations for both types of components. See more details below
      ```
    * If you install Longhorn by using `kubectl` to apply [the deployment YAML](https://raw.githubusercontent.com/longhorn/longhorn/v1.1.1/deploy/longhorn.yaml), you need to modify the taint tolerations section for Longhorn Manager, Longhorn UI, and Longhorn Driver Deployer.
     Then apply the YAMl files.
-   * If you install Longhorn using Helm, you can change the Helm values for `longhornManager.tolerations`, `longhornUI.tolerations`, `longhornDriver.tolerations` in the `values.yaml` file.
+   * If you install Longhorn using Helm, you can change the Helm values for `global.tolerations`, `longhornManager.tolerations`, `longhornUI.tolerations`, `longhornDriver.tolerations` in the `values.yaml` file before installing the chart.
     Then install the chart
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:51]: Please remove this line. The words are already part of the previous sentence.

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 16]
```diff
@@ -69,8 +78,17 @@ You need to set tolerations for both types of components. See more details below
...
+   * If you install Longhorn through Rancher 2.5.x, you must copy and paste the following parameters into the YAML on the Rancher UI (click **Edit as YAML** during the process of upgrad) to apply the value to all user-deployed components.
+     ```yaml
+       global:
+         tolerations:
+         - key: "key"
+           operator: "Equal"
+           value: "value"
+           effect: "NoSchedule"
+    ```
+   * You can also specify the tolerations for each user-deployed components:
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:91]:
```suggestion
   * You can also specify tolerations for each user-deployed component.
```

Comment (jillian-maroket) [reply content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:91]: I flagged this in my initial review.

File: content/docs/1.7.0/advanced-resources/deploy/node-selector.md [hunk 17]
```diff
@@ -54,8 +60,14 @@ You need to set node selector for both types of components. See more details bel
 1. Prepare
    * To ensure that your preferred settings are immediately applied, stop all workloads and detach all Longhorn volumes before applying it.
 
-2. Set node selector for user deployed components
-    * If you install Longhorn by Rancher UI, you need to click `Edit as YAML` and copy this values into the YAML then click upgrade:
+2. Set node selector for user-deployed components
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/node-selector.md:63]:
```suggestion
1. Set the node selector for user-deployed components (for example, Longhorn Manager, Longhorn Driver, and Longhorn UI).
```

File: content/docs/1.7.0/advanced-resources/deploy/node-selector.md [hunk 18]
```diff
@@ -69,10 +81,9 @@ You need to set node selector for both types of components. See more details bel
         ```
     * If you install Longhorn by using `kubectl` to apply [the deployment YAML](https://raw.githubusercontent.com/longhorn/longhorn/v1.1.1/deploy/longhorn.yaml), you need to modify the node selector section for Longhorn Manager, Longhorn UI, and Longhorn Driver Deployer.
       Then reapply the YAMl files.
-    * If you install Longhorn using Helm, you can change the Helm values for `longhornManager.nodeSelector`, `longhornUI.nodeSelector`, `longhornDriverDeployer.nodeSelector` in the `value.yaml` file.
-      Then do Helm upgrade the chart.
+    * If you install Longhorn using Helm, you can change the Helm values for `global.nodeSelector`, `longhornManager.nodeSelector`, `longhornUI.nodeSelector`, `longhornDriverDeployer.nodeSelector` in the `values.yaml` file, and then run `helm upgrade` to upgrade to the new version of the chart.
 
-3. Set node selector for system managed components
+3. Set node selector for system-managed components
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/node-selector.md:86]:
```suggestion
3. Set the node selector for system-managed components (for example, Instance Manager, Backing Image Manager, Share Manager, CSI Driver, and Engine Image).
```

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 19]
```diff
@@ -69,8 +78,17 @@ You need to set tolerations for both types of components. See more details below
 
    To ensure that your preferred settings are immediately applied, stop all workloads and detach all Longhorn volumes before configuring the settings.
 
-2. Set taint tolerations for user deployed components (Manager, UI, Driver Deployer)
-   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML:
+2. Set taint tolerations for user-deployed components (Manager, UI, Driver Deployer)
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:81]:
```suggestion
2. Set taint tolerations for user-deployed components (for example, Longhorn Manager, Longhorn Driver, and Longhorn UI).
```

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 20]
```diff
@@ -69,8 +78,17 @@ You need to set tolerations for both types of components. See more details below
...
+   * If you install Longhorn through Rancher 2.5.x, you must copy and paste the following parameters into the YAML on the Rancher UI (click **Edit as YAML** during the process of upgrad) to apply the value to all user-deployed components.
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:82]:
```suggestion
   * If you install Longhorn through Rancher 2.5.x, you must copy and paste the following parameters into the YAML on the Rancher UI (click **Edit as YAML** during the upgrade) to apply the value to all user-deployed components.
```

File: content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md [hunk 21]
```diff
@@ -93,10 +111,9 @@ You need to set tolerations for both types of components. See more details below
      ```
    * If you install Longhorn by using `kubectl` to apply [the deployment YAML](https://raw.githubusercontent.com/longhorn/longhorn/v1.1.1/deploy/longhorn.yaml), you need to modify the taint tolerations section for Longhorn Manager, Longhorn UI, and Longhorn Driver Deployer.
   Then reapply the YAMl files.
-   * If you install Longhorn using Helm, you can change the Helm values for `longhornManager.tolerations`, `longhornUI.tolerations`, `longhornDriver.tolerations` in the `values.yaml` file.
-  Then do Helm upgrade the chart.
+   * If you install Longhorn using Helm, you can change the Helm values for `global.tolerations`, `longhornManager.tolerations`, `longhornUI.tolerations`, `longhornDriver.tolerations` in the `values.yaml` file, and then run `helm upgrade` to upgrade to the new version of the chart.
 
-3. Set taint tolerations for system managed components
+3. Set taint tolerations for system-managed components
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md:116]:
```suggestion
3. Set taint tolerations for system-managed components (for example, Instance Manager, Backing Image Manager, Share Manager, CSI Driver, and Engine Image).
```

**Review by jillian-maroket — COMMENTED** [review]
@ChanYiLin The workaround you used is fine. I just noticed that additional changes were made to the text so I reviewed those and flagged items that were missed.

**Comment by ChanYiLin** [comment]
Hi @jillian-maroket 
Fixed, thank you!

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

File: content/docs/1.7.0/advanced-resources/deploy/node-selector.md [hunk 22]
```diff
@@ -10,12 +10,18 @@ In this case, you can set the node selector to restrict Longhorn to only run on
...
 # Setting up Node Selector for Longhorn
-Longhorn system contains user deployed components (e.g, Manager, Driver Deployer, UI) and system managed components (e.g, Instance Manager, Engine Image, CSI Driver, etc.)
+Longhorn consists of user-deployed components (for example, Longhorn Manager, Longhorn Driver, and Longhorn UI) and system-managed components (for example, Instance Manager, Backing Image Manager, Share Manager, CSI Driver, and Engine Image).
 You need to set node selector for both types of components. See more details below.
 
 ### Setting up Node Selector During installing Longhorn
-1. Set node selector for user deployed components (Manager, UI, Driver Deployer)
-   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML:
+1. Set the node selector for user-deployed components (for example, Longhorn Manager, Longhorn Driver, and Longhorn UI).
+   * If you install Longhorn through Rancher 2.5.x, you must copy and paste the following parameters into the YAML on the Rancher UI (click **Edit as YAML** during the installation) to apply the value to all user-deployed components.
```
Comment (innobead) [inline content/docs/1.7.0/advanced-resources/deploy/node-selector.md:18]: `Rancher 2.5.x` is quite old version. Let's mention Rancher w/o any specific version.

File: content/docs/1.7.0/advanced-resources/deploy/node-selector.md [hunk 23]
```diff
@@ -10,12 +10,18 @@ In this case, you can set the node selector to restrict Longhorn to only run on
...
+      ```yaml
+        global:
+          nodeSelector:
+            label-key1: "label-value1"
+      ```
+   * You can also specify the node selector for each user-deployed component.
```
Comment (innobead) [inline content/docs/1.7.0/advanced-resources/deploy/node-selector.md:24]: Should explain the setting will override the global setting instead of merging.

**Review by innobead — CHANGES_REQUESTED** [review]
Please fix the issues I mentioned for node selector and taint toleration. Besides, we need to backport this improvement to 1.6.3 and 1.5.6.

**Review by innobead — APPROVED** [review]

**Pull request merged**